addons/survey/models/survey_survey.py

```diff
@@ -568,6 +568,32 @@ def _get_number_of_attempts_lefts(self, partner, email, invite_token):
     # QUESTIONS MANAGEMENT
     # ------------------------------------------------------------
 
+    def _remove_invalid_conditional_questions(self, questions):
+        possible_trigger_questions = self.env['survey.question'].search([
+            ('is_page', '=', False),
+            ('question_type', 'in', ['simple_choice', 'multiple_choice']),
+            ('suggested_answer_ids', '!=', False),
+            ('survey_id', '=', self.id)
+        ])
```

Comment:
Could the highest `sequence` of `questions`  value be used to make sure we do not request a potentially large number of questions that are surely not possible triggers?